client/src/components/install.jsx

```diff
@@ -8,7 +8,7 @@ function install(props) {
   const { info } = props;
 
   return (
-    <Container>
+    <>
```

Comment:
Probably invalid XML.  Maybe just comment out?  Can clients override style "display: none"?

Reply:
Man.. I had addon interference in my VSC... really messed me up for a minute there lol but then I discovered how to use Prettier, this next PR should fix any formatting issues.

Reply:
> Can clients override style "display: none"?

I don't think so, when node.js compiles everything it removes all comment lines in the final client .js build files to make it very efficient for production.